src/applications/personalization/profile360/vet360/actions/transactions.js

```diff
@@ -66,7 +66,8 @@ export function refreshTransaction(transaction, analyticsSectionName) {
         transaction
       });
 
-      const transactionRefreshed = isVet360Configured() ? await apiRequest(`/profile/status/${transactionId}`) : await localVet360.updateTransaction(transactionId);
+      const route = _route || `/profile/status/${transactionId}`;
+      const transactionRefreshed = isVet360Configured() ? await apiRequest(route) : await localVet360.updateTransaction(transactionId);
 
       if (isSuccessfulTransaction(transactionRefreshed)) {
```

Comment:
I didn't know this, but there is a [special route](https://dsva.slack.com/archives/C909ZG2BB/p1534514851000100?thread_ts=1534434974.000100&cid=C909ZG2BB) for checking the initialization transaction status.